src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -34,77 +34,71 @@ export class IssueQuickAccess extends PickerQuickAccessProvider<IPickerQuickAcce
 	}
 
 	protected override _getPicks(filter: string): Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem> | Promise<Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem>> | null {
-		if (this.configurationService.getValue<string>('extensions.experimental.quickAcessExtensions')) {
-			const issuePicks: Array<IPickerQuickAccessItem | IQuickPickSeparator> = [];
-			const extensionIdList: Array<string> = [];
-
-			// add regular open issue reporter button
-			const createTerminalLabel1 = this.productService.quality === 'stable' ? localize("workbench.action.openIssueReporter", "Visual Studio Code") : localize("workbench.action.openIssueReporterInsiders", "Visual Studio Code: Insiders");
-			issuePicks.push({
-				label: `$(plus) ${createTerminalLabel1}`,
-				ariaLabel: createTerminalLabel1,
-				accept: () => this.commandService.executeCommand('workbench.action.openIssueReporter', { issueSource: IssueSource.VSCode })
-			});
-
-			if (issuePicks.length > 0) {
-				issuePicks.push({ type: 'separator' });
+		if (!this.configurationService.getValue<boolean>('extensions.experimental.issueQuickAccess')) {
```

Comment:
If I have the setting disabled, does the quick access provider still exist but it does nothing? I would think that this setting enabled the provider showing up at all.